1_basics.ipynb

```diff
@@ -254,14 +502,28 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
     "    <b>次の学習</b>\n",
     "    <p>HSRに搭載されているセンサの情報を取得し、台車を制御して壁の手前まで動かしてみます。</p>\n",
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>What's next</b>\n",
```

Comment:
Here, because it is a question (and it feels like a title to a section), the punctuation (a question mark) is required:

    "    <b>What's next?</b>\n",